js/about/bookmarks.js

```diff
@@ -201,9 +215,22 @@ class AboutBookmarks extends React.Component {
       selectedFolderId: id
     })
   }
+  onChangeSearch (evt) {
+    console.log('search term changed')
+    this.setState({
+      search: evt.target.value
+    })
+  }
+  searchedBookmarks (searchTerm, bookmarks) {
+    return bookmarks.filter((bookmark) => {
+      const title = bookmark.get('customTitle') || bookmark.get('title')
+      return title.match(new RegExp(searchTerm, 'gi'))
```

Comment:
Pls search all of title, customTitle and location.  For example if someone does www.google.com it should find a match